### ISSUE TITLE
Warning when installing QeApp

### ISSUE THREAD

**Issue by danielhollas** [aiidalab/aiidalab#455]
I saw the following warning popup when installing QeApp in appstore

![image](https://github.com/user-attachments/assets/58f6153d-0ceb-44d4-89ed-67d59bbc4a3c)

It comes from this code.
https://github.com/aiidalab/aiidalab/blob/b39a76df225b9f1b1a290779ac72b95edd4e8218/aiidalab/app.py#L213

Not sure if this is a race condition while the app is being installed, or something more.

**Comment by ayushjariyal** [comment]
Hi @danielhollas , I want to try this issue. Is it still open?

**Comment by danielhollas** [comment]
@ayushjariyal I don't think this is a good issue to work on since I think this is an intermittent issue that we don't have reproduction yet. I'll go through the issues on this repo and tag you on those that I think might be good to work on (#432 is a good example!)

**Comment by ayushjariyal** [comment]
Thank you @danielhollas , I’m really excited to work on these issues! Looking forward to your suggestions and happy to contribute wherever I can. 😊
